src/platform/forms-system/src/js/fields/ObjectField.jsx

```diff
@@ -180,9 +178,10 @@ class ObjectField extends React.Component {
 
     // description and title setup
     const showFieldLabel = uiOptions.showFieldLabel;
-    const fieldsetClassNames = classNames(uiOptions.classNames, {
-      'vads-u-margin-y--2': !environment.isProduction(),
-    });
+    const fieldsetClassNames = classNames(
+      uiOptions.classNames,
+      'vads-u-margin-y--2',
+    );
```

Comment:
Related PR: https://github.com/department-of-veterans-affairs/vets-website/pull/19535